vowpalwabbit/core/src/reductions/interaction_ground.cc

```diff
@@ -23,6 +24,46 @@ using namespace VW::LEARNER;
 using namespace VW::config;
 using namespace VW::reductions;
 
+namespace VW
+{
+namespace reductions
+{
+namespace model_utils
```

Comment:
May be useful to add a couple simple run_test's using -i/-f and --readable_model to make it clear what igl is saving